pmdk.spec

```
@@ -17,16 +17,16 @@
 
 %global major 2
 %global minor 0
-%global bugrelease 0
+%global bugrelease 1
 #%%global prerelease rc1
 %global buildrelease 1
 
 %global _hardened_build 1
 
 # by default build with ndctl, unless explicitly disabled
-%bcond_without ndctl
+%bcond_with ndctl
```

I checked the actual build and it seems it's still building without _ndctl_:

```
+ make EXTRA_CFLAGS=-Wno-error NORPATH=1 BUILD_EXAMPLES=n BUILD_BENCHMARKS=n NDCTL_ENABLE=n check
```

Upon [re-reading about `%bcond_with[out]`](https://rpm-software-management.github.io/rpm/manual/conditionalbuilds.html), as confusing at it seems we want to use:

```suggestion
%bcond_without ndctl
```

That does make sense when you consider the hint that the explanation in the URL above:
> These macros create command line options.

So what we are trying to do here is create a `--without-ndctl` `rpmbuild` option to disable the default behaviour of building with _ndctl_.

Apologies for any previous mis-direction I may have given in this regard.

---

Done